ui/imports/shared/popups/keycard/states/ManageAccounts.qml

```diff
@@ -100,18 +107,110 @@ Item {
         anchors.rightMargin: Style.current.xlPadding
         spacing: Style.current.padding
 
+        StatusStepper {
+            id: stepper
+            Layout.preferredWidth: Constants.keycard.general.keycardNameInputWidth
+            Layout.alignment: Qt.AlignCenter
+            title: qsTr("Account %1 of %2")
+            titleFontSize: Constants.keycard.general.fontSize2
+        }
+
         StatusBaseText {
             id: title
             Layout.alignment: Qt.AlignCenter
             font.weight: Font.Bold
         }
 
+        Rectangle {
+            id: accountDetails
+            Layout.preferredWidth: Constants.keycard.general.keycardNameInputWidth
+            Layout.alignment: Qt.AlignCenter
+            height: Style.current.xlPadding * 2
+            color: "transparent"
+            border.color: Theme.palette.baseColor2
+            border.width: 1
+            radius: Style.current.halfPadding
+
+            RowLayout {
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.left: parent.left
+                anchors.right: parent.right
+                anchors.leftMargin: Style.current.padding
+                anchors.rightMargin: Style.current.padding
+
+                ColumnLayout {
+                    StatusBaseText {
+                        text: d.observedImportingAccountShortAddress
+                        wrapMode: Text.WordWrap
+                        font.pixelSize: Constants.keycard.general.fontSize2
+                        color: Theme.palette.baseColor1
+                    }
+
+                    StatusBaseText {
+                        text: root.sharedKeycardModule.keyPairHelper.observedAccount.path
+                        wrapMode: Text.WordWrap
+                        font.pixelSize: Constants.keycard.general.fontSize2
+                        color: Theme.palette.baseColor1
+                    }
+                }
+
+                Item {
+                    Layout.fillWidth: true
+                    Layout.preferredHeight: parent.height
+                }
+
+                ColumnLayout {
+                    StatusBaseText {
+                        text: {
+                            if (Global.appMain) {
+                                return "Balance: %1%2".arg(SharedStore.RootStore.currencyStore.currentCurrencySymbol)
+                                .arg(Utils.toLocaleString(root.sharedKeycardModule.keyPairHelper.observedAccount.balance.toFixed(2), appSettings.locale, {"model.account.currency": true}))
```

Comment:
this will need to change to match the new currency formatting scheme. Let's see which PR gets merged first 😈